planning/route_handler/include/route_handler/route_handler.hpp

```diff
@@ -163,8 +239,6 @@ class RouteHandler
   bool isBijectiveConnection(
     const lanelet::ConstLanelets & lanelet_section1,
     const lanelet::ConstLanelets & lanelet_section2) const;
-  bool getPreviousLaneletWithinRoute(
```

Comment:
@zulfaqar-azmi-t4 
can you stay this line still to make less diff ?

Reply:
For the you moved to public is to use  this in for feature PR and it needs to be public. I resolve this issue.